integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/utils/AbstractWebIT.java

```diff
@@ -15,7 +15,7 @@
 public class AbstractWebIT extends AbstractIT {
   protected static final Logger LOG = LoggerFactory.getLogger(AbstractWebIT.class);
   protected static WebDriver driver;
-  protected static final long MAX_IMPLICIT_WAIT = 60;
+  protected static final long MAX_IMPLICIT_WAIT = 10;
```

Comment:
Why need reduce wait time？

Reply:
When searching for elements, this implicit wait time is too long, which will increase the time it takes for testing.